### PR TITLE
Refactor unimplemented opcodes

### PR DESCRIPTION
**Pull request by MarcosNicolau**
For those opcodes that are not yet implemented, instead of panicking the process, stop vm execution as panic.

### CODE REVIEW

File: src/op_handlers/unimplemented.rs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+use crate::eravm_error::{EraVmError, OpcodeError};
+use crate::{opcode::Opcode, state::VMState};
+
+pub fn unimplemented(_vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmError> {
+    eprintln!("Unimplemented instruction: {:?}!", opcode.variant);
```
Comment (jorbush) [inline src/op_handlers/unimplemented.rs:5]: why is this `print` necessary?

File: src/op_handlers/context.rs [hunk 2]
```diff
@@ -76,7 +76,7 @@ pub fn set_context_u128(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmEr
     Ok(())
 }
 
-pub fn aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
+pub fn _aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
```
Comment (jorbush) [inline src/op_handlers/context.rs:79]: Seeing the new implementation of `unimplemented` this no longer seems to be necessary

**Review by jorbush — CHANGES_REQUESTED** [review]

File: src/op_handlers/unimplemented.rs (hunk 1 above)
Comment (MarcosNicolau) [reply src/op_handlers/unimplemented.rs:5]: Just to make sure we can easier spot where the execution ended

File: src/op_handlers/context.rs (hunk 2 above)
Comment (MarcosNicolau) [reply src/op_handlers/context.rs:79]: I left it for when we have to implement it

**Review by jorbush — APPROVED** [review]
LGTM

**Review by Oppen — APPROVED** [review]

**Pull request merged**